src/internal/operators/refCount.ts

```diff
@@ -60,20 +60,18 @@ import { Observable } from '../Observable';
  */
 export function refCount<T>(): MonoTypeOperatorFunction<T> {
   return function refCountOperatorFunction(source: ConnectableObservable<T>): Observable<T> {
-    return source.lift(new RefCountOperator(source));
+    return source.lift(new RefCountOperator());
   } as MonoTypeOperatorFunction<T>;
 }
 
 class RefCountOperator<T> implements Operator<T, T> {
-  constructor(private connectable: ConnectableObservable<T>) {
```

Comment:
Removing the constructor is also a breaking change.

Reply:
I'm fine with this. AFAICT, this is not a breaking change as `RefCountOperator` is not exported. And if someone has reached into the internals to get it and this breaks them, then that's on them for being naughty.